packages/detekt/package.yaml

```
@@ -0,0 +1,24 @@
+---
+name: detekt
+description: |
+  A static code analyzer for Kotlin.
+homepage: https://detekt.dev/
+licenses:
+  - Apache-2.0
+languages:
+  - Kotlin
+categories:
+  - Linter
+
+source:
+  id: pkg:github/detekt/detekt@v1.23.3
+  asset:
+    - target: unix
+      file: detekt-cli-{{ version | strip_prefix "v" }}.zip
+      bin: detekt-cli-{{ version | strip_prefix "v" }}/bin/detekt-cli
+    - target: win
+      file: detekt-cli-{{ version | strip_prefix "v" }}.zip
+      bin: detekt-cli-{{ version | strip_prefix "v" }}/bin/detekt-cli
```

I believe this should be the .bat file, no?

```suggestion
      bin: detekt-cli-{{ version | strip_prefix "v" }}/bin/detekt-cli.bat
```

---

You are right :sob: